mobify/platform-scaffold
# top align field and button to avoid breaking layout with error

invalid email breaks the newsletter layout

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1388

## Changes
- top align field and button

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- (how to access the new / changed functionality -- fixtures, URLs)
- submit 'test' in newsletter to see the error
